libs/shared/flagd-core/package.json

```diff
@@ -1,12 +1,13 @@
 {
   "name": "@openfeature/flagd-core",
-  "version": "0.0.1-alpha",
+  "version": "0.1.0",
   "scripts": {
     "publish-if-not-exists": "cp $NPM_CONFIG_USERCONFIG .npmrc && if [ \"$(npm show $npm_package_name@$npm_package_version version)\" = \"$(npm run current-version -s)\" ]; then echo 'already published, skipping'; else npm publish --access public; fi",
     "current-version": "echo $npm_package_version"
   },
   "dependencies": {
-    "@openfeature/server-sdk": ">=1.6.0",
+    "@types/imurmurhash": "^0.1.3",
```

Comment:
I think this should be a dev dependency and `imurmurhash` should be a dependency.

FYI @toddbaert